test/objects_test.rb

```diff
@@ -51,4 +51,18 @@ def test_event
     event_json = '{"ts":123,"action":"create","resource":{"@ref":"classes/frogs/123"}}'
     assert_equal event_json, to_json(Event.new(123, 'create', @ref))
   end
+
+  def test_ts
+    test_ts = Time.at(0).utc
+    test_ts_json = '{"@ts":"1970-01-01T00:00:00.000000000Z"}'
+    assert_equal test_ts_json, FaunaJson.to_json(test_ts)
```

Comment:
I think it would make more sense to keep using the `to_json` defined here but change that method to call the one at `FaunaJson`.